core/src/integrationTest/java/tech/pegasys/web3signer/core/jsonrpcproxy/DefaultTestBase.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 ConsenSys AG.
```

Comment:
Does this matter?

Reply:
If creating the file as new in web3signer would set it as this year then maybe, but probably not an issue either way